pages/price-feeds/create-your-first-pyth-app/evm/part-2.mdx

```diff
@@ -1,14 +1,25 @@
+import { Callout } from "nextra/components";
+
 # Deploy your Pyth App
 
-In this part of the tutorial, we will deploy a Solidity contract that uses Pyth price feeds.
-This is continuation of the [Create your first Pyth app on EVM](./part1) of the tutorial.
-If you haven't completed that part yet, please do so before continuing.
+In this part of the tutorial, we will learn to deploy the `MyFirstPythContract.sol` contract and fetch the latest price from Pyth network.
+
+This part of the tutorial will conver the following:
+
+- Deploy the contract on OP Sepolia testnet.
+- Interact with the contract from the command line.
+- Update and Fetch the price from the contract using [pyth-evm-js](https://github.com/pyth-network/pyth-crosschain/tree/main/target_chains/ethereum/sdk/js).
```

Comment:
two comments:
* "Fetch" shouldn't be capitalized
* I think the order of the verbs should reflect the order of the actions, so it should be "fetch and update"

Reply:
Since we are referring to price here, we should keep it `update and fetch`.